python/langsmith/evaluation/_runner.py

```diff
@@ -1793,16 +1794,20 @@ def _wrap(evaluator: SUMMARY_EVALUATOR_T) -> SUMMARY_EVALUATOR_T:
 
         @functools.wraps(evaluator)
         def _wrapper_inner(
-            runs: Sequence[schemas.Run], examples: Sequence[schemas.Example]
+            runs: Sequence[schemas.Run],
+            examples: Sequence[schemas.Example],
+            evaluation_results: Sequence[schemas.ExperimentResultRow],
```

Comment:
should we just pass in the EvaluationResults not the whole row?

Reply:
I think it's easier to associate with which example the result came from - which I imagine might be useful? Not 100% sure, up to you just lmk what to do